dask/dataframe/groupby.py

```diff
@@ -1290,6 +1290,12 @@ def _cum_agg(self, token, chunk, aggregate, initial):
         graph = HighLevelGraph.from_collections(name, dask, dependencies=dependencies)
         return new_dd_object(graph, name, chunk(self._meta), self.obj.divisions)
 
+    def _compute(self):
```

Comment:
The idea is to match the compute method on the base class so that if someone accidentally calls `gb.compute(scheduler="threads") for instance, they will get the `NotImplementedError`